rustls/src/verify.rs

```diff
@@ -522,26 +522,48 @@ fn prepare<'a, 'b>(
     Ok((cert, intermediates, trustroots))
 }
 
+/// A DER encoded Certificate Revocation List (CRL).
+pub struct CertRevocationList(pub Vec<u8>);
```

Comment:
Perhaps `UnparsedCertRevocationList`? Or something else to designate that nothing is checking the contents of this.

Reply:
Renamed.